subject_files_status_for_dpdash.py

```diff
@@ -59,7 +62,7 @@ def get_summary_from_phoenix(phoenix_dir: Path) -> pd.DataFrame:
     return df
     
 
-df = get_summary_from_phoenix(pronet_phoenix_dir)
+df = get_summary_from_phoenix([pronet_phoenix_dir, pronet_phoenix_dir])
```

Comment:
Sorry, I spoke too quickly before. You idea looks fine. But I think you made a typo in the second element. It should be `prescient_phoenix_dir`.

Reply:
Again, for creating separate dashboards for the two networks, we should use separate output directories. Otherwise, my next script `project_files_status_for_dpdash.py` won't work. I shall propose my idea soon and then maybe you can review.